<launch>
  <arg name="ASR_Topic" />
  <arg name="NLU_Topic" />

  <node pkg="iflytek_asr" type="iflytek_asr_node" name="iflytek_asr_node" output="screen">
    <param name="app_id" type="String" value="57207923" />

    <param name="asr_topic" value="$(arg ASR_Topic)" />
    <param name="nlu_topic" value="$(arg NLU_Topic)" />
  </node>
</launch>
